packages/components/app/styles/components/tooltip.scss

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// TOOLTIP & TOOLTIP-BUTTON
+//
+
+@use "../mixins/focus-ring" as *;
+
+:root {
+  // Dimensions:
+  --token-tooltip-max-width: 280px;
+
+  // Focus highlight position
+  --token-tooltip-tab-focus-offset: -2px;
+
+  // Colors:
+  --token-tooltip-surface-primary: var(--token-color-palette-neutral-700);
+}
+
+.hds-tooltip-button {
+  @include hds-focus-ring-with-pseudo-element(
+    $top: var(--token-tooltip-tab-focus-offset),
+    $right: var(--token-tooltip-tab-focus-offset),
+    $bottom: var(--token-tooltip-tab-focus-offset),
+    $left: var(--token-tooltip-tab-focus-offset),
+  );
+  margin: 0;
+  padding: 0;
+  color: inherit;
+  background: transparent;
+  border: none;
+
+  > .flight-icon { display: block; }
+}
+
+// this is the default html button display
+.hds-tooltip-button--is-inline {
+  display: inline-block;
+}
+
+.hds-tooltip-button--is-block {
+  display: flex;
+}
+
+.tippy-box[data-theme~="hds"] {
+  padding: 8px 12px;
+  color: var(--token-color-foreground-high-contrast);
+  font-weight: var(--token-typography-font-weight-regular);
+  font-size: var(--token-typography-body-200-font-size);
+  font-family: var(--token-typography-body-200-font-family);
+  line-height: var(--token-typography-body-200-line-height);
+  background-color: var(--token-tooltip-surface-primary);
+  border-radius: 5px;
+  box-shadow: var(--token-elevation-higher-box-shadow);
+  transition-property: transform, visibility, opacity;
+
+  &[data-animation="fade"][data-state="hidden"] {
+    opacity: 0;
+  }
+
+  &[data-inertia][data-state="visible"] {
+    transition-timing-function: cubic-bezier(0.54, 1.5, 0.38, 1.11);
```

Comment:
design token? (see how it's declared in the design tokens for `Tabs`, as an example)